08-conclusion.md

```diff
@@ -28,12 +28,12 @@ papers.
 > Add new rules, update existing rules, and add new macros to:
 > 
 > * Create `.png` files from `.dat` files using `plotcount.py`.
-> * Add the script and `.png` files to the archive.
+> * Add the script and `.png` files as dependencies of the `results.txt` file to make sure they are (re)created when needed.
```

Comment:
It feels wrong to add the `.png` files as dependent on `results.txt` as this is artificial. It'd be cleaner to just extend `all` to build both `results.txt` and the `.png` files.